colour/volume/macadam_limits.py

```diff
@@ -142,3 +143,197 @@ def is_within_macadam_limits(
     simplex = np.where(simplex >= 0, True, False)
 
     return simplex
+
+
+def macadam_limits(target_brightness, illuminant=()):
```

Comment:
This initial docstring is just too long. We should also not constrain the user to a particular spectral domain, the definition should handle that for him according to the passed `CMFS`.

Reply:
I do not exactly understand what ist ment, but I did a more sophisticated solution in the actual code.